src/overlays/actors/ovl_Eff_Stk/z_eff_stk.c

```diff
@@ -28,16 +28,87 @@ ActorInit Eff_Stk_InitVars = {
     (ActorFunc)EffStk_Draw,
 };
 
-#endif
+void func_80BF0DE0(EffStk* arg0, PlayState* arg1);
 
-extern UNK_TYPE D_06009F60;
+void EffStk_Init(Actor* thisx, PlayState* play) {
+    EffStk* this = (EffStk*)thisx;
+    Actor_SetScale(&this->actor, 0.2f);
+    this->actionFunc = func_80BF0DE0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Stk/EffStk_Init.s")
+void EffStk_Destroy(Actor* thisx, PlayState* play) {
+    EffStk* this = (EffStk*)thisx;
+}
```

Comment:
Is this variable required? Or can the function be left empty? 
```suggestion
void EffStk_Destroy(Actor* thisx, PlayState* play) {
}
```